apps/onlyoffice.sh

```diff
@@ -70,29 +70,55 @@ else
    exit 1
 fi
 
-# Check if 443 is open using nmap, if not notify the user
+# Check to see if user already has nmap installed on their system
+
+if [ "$(dpkg-query -s nmap 2> /dev/null | grep -c "ok installed")" == "1" ]
+then
+    NMAPSTATUS=preinstalled
+else
+    NMAPSTATUS=notinstalled
```

Comment:
you don't need `notinstalled` as that variable isn't used. So you can remove the `else`. :)